<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Youtube Chat Statistics</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="fileNameLabel">
        <property name="text">
         <string>No file chosen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fileOpenButton">
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string>Select chat file</string>
        </property>
        <property name="text">
         <string>Choose file</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Chat</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTextEdit" name="chatTextEdit">
            <property name="documentTitle">
             <string>Aboba</string>
            </property>
            <property name="placeholderText">
             <string>You can place chat recording here</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Statistics</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="overviewTab">
             <attribute name="title">
              <string>Overview</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>1</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>1</number>
              </property>
              <item>
               <widget class="QStackedWidget" name="overviewStackedWidget">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="overviewBlankPage">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="overviewBlankPageLabel">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#888a85;&quot;&gt;Place chat recording first!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="overviewPage">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="userTab">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <attribute name="title">
              <string>User</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>1</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>1</number>
              </property>
              <item>
               <widget class="QStackedWidget" name="userStackedWidget">
                <widget class="QWidget" name="userBlankPage">
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#888a85;&quot;&gt;Double click to user in overview page!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="userPage"/>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="statusTip">
      <string>Open recent chat recordings</string>
     </property>
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit Youtube Chat Statistics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open chat recording</string>
   </property>
   <property name="statusTip">
    <string>Open chat recording</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
